tutorials/speaker_tasks/Speaker_Identification_Verification.ipynb

```diff
@@ -137,7 +137,7 @@
                 "id": "BhWVg2QoDhL3"
             },
             "source": [
-                "Let's look at the first 3 lines of scp file for train."
+                "Let's look at the first 3 lines of text file for train."
```

Comment:
Probably this is refactored scp->txt, but this is better to be "filelist text" file